cnxpublishing/tests/views/test_admin.py

```diff
@@ -97,6 +97,101 @@ def test(self):
                 ]}, resp_data)
 
 
+class PrintStyleViewsTestCase(unittest.TestCase):
+    maxDiff = None
+
+    @classmethod
+    def setUpClass(cls):
+        cls.settings = integration_test_settings()
+        from cnxpublishing.config import CONNECTION_STRING
+        cls.db_conn_str = cls.settings[CONNECTION_STRING]
+        cls.db_connect = staticmethod(db_connection_factory())
+
+    def setUp(self):
+        self.config = testing.setUp(settings=self.settings)
+        self.config.include('cnxpublishing.tasks')
+        self.config.add_route('get-content', '/contents/{ident_hash}')
+        self.config.add_route('admin-content-status-single',
+                              '/a/content-status/{uuid}')
+        self.config.add_route('admin-print-style-single',
+                              '/a/print-style/{style}')
+        self.config.add_route('get-resource', '/resources/{hash}')
```

Comment:
It's hard to tell from this diff and PR, since I reorganized the commits and rebased, but this is a combo of @sisiz and me. :-) This is pretty much a clone of the ContentViewsTestCase. I'll poke at it a bit and see how much goes away w/ the include. Seems reasonable to include the condig for views when testing views, and it's much DRYer

Reply:
fixed